src/index.js

```diff
@@ -32,7 +32,7 @@ const yaml = require('yamljs');
 const util = require('util');
 const router = require('@internal/router');
 const https = require('https');
-
+const cors = require('@koa/cors');
```

Comment:
why do we need CORS?

Reply:
To allow the the browser to interact with the backend.
Curl / postman will not complain about missing CORS, but the browser runs on a different port, so it requires CORS.